js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gadgetinc/dateilager",
-  "version": "0.9.3",
+  "version": "0.9.5-pre.018867c",
```

Comment:
Just make sure you run `make prerelease-reset` before merging into master.

Reply:
thx